__init__.py

```diff
@@ -0,0 +1,11 @@
+from spellcheck import *
+import sys
+
+def main():
+	spellchk = SpellCheck('/usr/share/dict/words')

```

Comment:
use 4 spaces instead of tabs.

Reply:
sure , will keep that in mind.

Reply:
@bhanuvrat why ?use tabs instead of spaces?